composer.json

```diff
@@ -3,14 +3,19 @@
     "type": "typo3-cms-extension",
     "description": "This extension extends the TYPO3 translation handling by translation records that can be edited by backend users. In this way backend users are able to translate labels without having access to the language files.",
     "license": ["GPL-2.0-or-later"],
+    "version": "3.0.0",
```

Comment:
omit version number from composer file because it will be taken from git tag.

we can also use 2.1.0 because there are no breaking changes from my point of view